README.md

```diff
@@ -411,6 +411,11 @@ set. The following commands are available:
 - `clojure-ts-convert-collection-to-vector`
 - `clojure-ts-convert-collection-to-set`
 
+### Add arity to a function
```

Comment:
Looking at the code, this works with more than just functions. (e.g. macros, multimethods, etc)

Reply:
I would probably say it works for functions and macros. All supported forms except macros are technically functions.  I'll extend the readme.